lib/ex_crypto.ex

```diff
@@ -36,6 +36,33 @@ defmodule ExCrypto do
     {kind, Exception.normalize(kind, error), System.stacktrace}
   end
 
+  @aes_block_size 16
+  @iv_bit_length 128
+
+  defp key_and_iv_for_algorithm(raw_key, raw_iv, algorithm) do
```

Comment:
We'll need to be explicit about if we are dealing with base64 encoding vs binaries. I've seen people get pretty confused about which to use. All of the length requirements for crypto refer to the raw bytes, and have nothing to do with the encoding.  You can feed a base64 encoded byte string sometimes, but it will just be used as raw bytes w/o any regard for what the decoded value would be.